EU4ToVic3/Source/V3World/EconomyManager/EconomyManager.h

```diff
@@ -89,34 +94,44 @@ class EconomyManager
 	[[nodiscard]] double calculateStateTraitMultiplier(const std::shared_ptr<SubState>& subState) const;
 	[[nodiscard]] double getDensityFactor(double perCapitaDev) const;
 
+	void establishBureaucracy(const std::shared_ptr<Country>& country, const std::map<std::string, Law>& lawsMap, const Vic3DefinesLoader& defines) const;
```

Comment:
```suggestion
	void establishBureaucracy(const Country& country, const std::map<std::string, Law>& lawsMap, const Vic3DefinesLoader& defines) const;
```

You're not taking (shared) ownership of the country, so just take a reference to it.